boot/zephyr/Kconfig

```diff
@@ -174,7 +174,7 @@ config BOOT_MAX_IMG_SECTORS
 
 config BOOT_ERASE_PROGRESSIVELY
 	bool "Erase flash progressively when receiving new firmware"
-	default y if SOC_NRF52840
+	default y if SOC_NRF52840 || SOC_NRF9160
```

Comment:
What the reason for enabling this for all nRF9160?

Reply:
This avoids a ~10 second delay at the beginning of the flashing process. Given that is saves this much time in a typical use case, I considered it a sensible default.